docs/content/docs/test/Runner.md

```diff
@@ -17,48 +17,203 @@ It uses an internal TMS (test management system). We are working on support othe
 
 ## Getting started
 
-TBD
-
-## Configuration
-
-A configuration is done in gradle script:
+1. {{< expand "Apply the gradle plugin" >}}
```

Comment:
I've used the `expand` widget. It makes documentation simpler to read
Expanded
![Expanded](https://user-images.githubusercontent.com/8560287/79751150-c6727680-831a-11ea-81b1-027364a73510.png)
Collapsed
![Collapsed](https://user-images.githubusercontent.com/8560287/79751148-c5414980-831a-11ea-98fb-633f77aba827.png)

Reply:
> It makes documentation simpler to read

On the contrary, I have to click on _each_ paragraph because I don't know what happens exactly and how to check it. When I see an implementation in the text I can easily skip it, I don't see the problem.

Compare with this one good tutorial: 
https://www.jetbrains.com/help/idea/creating-and-running-your-first-java-application.html
Imagine only titles. For instance, "Add the main() method using live templates".

I think than collapsed by default is ok in references but not in tutorials and how-tos.

Reply:
I propose to extract "Getting started" tutorial to a single page.
It will eliminate this contradiction.

Reply:
I think extracting to a single page depends on creating a new structure. It looks more consistent now. Do you know does expand have default state or it can be only collapsed?

Reply:
> does expand have default state or it can be only collapsed?

I don't see this ability: [shortcodes/expand.html](https://github.com/alex-shpak/hugo-book/blob/ba5d38ad4427bcda21a4cc57689464ed58d8b9a0/layouts/shortcodes/expand.html)

Reply:
I've created issue https://github.com/alex-shpak/hugo-book/issues/205